src/PanthereTestCase.php

```diff
@@ -65,24 +66,29 @@
      */
     protected static $panthereClient;
 
-    public static function tearDownAfterClass()
+    /**
+     * @var bool
+     */
+    protected static $stopServerOnTeardown = true;
```

Comment:
should use `private` and `self::` to access it. The listener always call it on PanthereTestCase so it does not make sense to allow child classes to override the property to get separate storages.

Reply:
Yep, wanted to do it, seems like I forgot :sweat_smile: , nice catch thanks!

Reply:
Done!